tests/ha_master/run.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
seems that we can combine `ha_master` test and `ha`, we can do this later

Reply:
and need to update https://github.com/pingcap/dm/blob/master/tests/others_integration.txt